### ISSUE TITLE
r-bot should read git tag instead of github release

### ISSUE THREAD

**Issue by TomasTomecek** [user-cont/release-bot#169]
```
Traceback (most recent call last):
  File "/opt/app-root/src/.local/bin/release-bot", line 11, in <module>
    sys.exit(main())
  File "/opt/app-root/src/.local/lib/python3.7/site-packages/release_bot/releasebot.py", line 302, in main
    rb.run()
  File "/opt/app-root/src/.local/lib/python3.7/site-packages/release_bot/releasebot.py", line 274, in run
    self.make_new_github_release()
  File "/opt/app-root/src/.local/lib/python3.7/site-packages/release_bot/releasebot.py", line 200, in make_new_github_release
    if Version.coerce(latest_release) >= Version.coerce(self.new_release['version']):
  File "/opt/app-root/src/.local/lib/python3.7/site-packages/semantic_version/base.py", line 140, in coerce
    % version_string
ValueError: Version string lacks a numerical component: 'The very first one'
```

because humans can title releases with funny names instead of the semver string, therefore release bot should get the associated tag name instead of github release title.

**Comment by shresthagrawal** [comment]
I am working on it!

**Comment by shresthagrawal** [comment]
What if people have funny names for the tags also ?

**Comment by TomasTomecek** [comment]
we should beat them then!

okay, jokes aside, yes that's also a normal case when people use specific tagging scheme, so probably make this configurable by specifying a regex which would get us from an upstream release to a git tag; but, this issue is really about just comparing two versions for the bot to know if there is some work to be done

**Comment by shresthagrawal** [comment]
please close this 
solved by #186
